Makefile

```diff
@@ -13,16 +13,16 @@ DOCKER_RUN_CMD ?= docker run \
 		  --rm \
 		  -it \
 		  -v /var/run/docker.sock:/var/run/docker.sock \
-		  -v ${PWD}:/workspace 
+		  -v ${PWD}:/workspace
 
 %:
 ifeq ($(MAKE_IN_DOCKER), 1)
 # Build builder image
-	@$(DOCKER_BUILD_CMD) -t $(DOCKER_BUILDER_IMAGE):$(DOCKER_BUILDER_TAG) - < tools/Dockerfile.builder
+	@$(DOCKER_BUILD_CMD) -t $(DOCKER_BUILDER_IMAGE):$(DOCKER_BUILDER_TAG) -f tools/Dockerfile.builder .
```

Comment:
please revert this, the tools/builder container is not supposed to be based on any host context

Reply:
I was wondering if we can send tools/install-python-3.8.sh from host to Dockerfile.builder like what istio did: https://github.com/istio/tools/blob/master/docker/build-tools/Dockerfile#L845